### PR TITLE
Rename logging_api directory to perpheral_api

### LINKED ISSUE #924: Rename Logging Server to Peripherals
This is just changing the [`api/logging_api/`](https://github.com/SCE-Development/Core-v4/blob/c66f3d27df2af08135b20401a0ba5dcc273da7c9/api/logging_api/server.js) directory to be `api/peripheral_api`. The reasoning being that it now is in charge of communicating with our peripherals (Printer/LED Sign). It also holds error logs... should we move them somewhere else?

Just rename the folder.

### PR DESCRIPTION
**Pull request by allen-tran**
Resolves #924

Renamed directory and all instances of `logging_api` or `LOGGING_API` to `peripheral_api`.

### CODE REVIEW

**Review by ajha7 — APPROVED** [review]

**Review by evanugarte — CHANGES_REQUESTED** [review]
- [ ] In the `config.example.jsons` update any instance of `loggingapi` to `peripheral_api`
- [ ] change the instantiation of the server in [api/peripheral_api/server.js:12](https://github.com/SCE-Development/Core-v4/blob/99062446765bae66f2b76dd3e0c6d4934da9f224/api/peripheral_api/server.js#L12)
to:
```js
  const peripheralServer = new SceHttpServer(API_ENDPOINTS, 8081, '/peripheral_api/');
  peripheralServer.initializeEndpoints().then(() => {
    peripheralServer.openConnection();
  });
```

**Review by neelp03 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "api-test": "mocha",
     "frontend-test": "mocha --require @babel/register --require ignore-styles test/frontend/*.js",
-    "start": "react-scripts start",
+    "start": "ESLINT_NO_DEV_ERRORS=true react-scripts start",
```
Comment (evanugarte) [inline package.json:8]:
```suggestion
    "start": "react-scripts start",
```

File: src/APIFunctions/2DPrinting.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:1]:
```suggestion
```

File: src/APIFunctions/2DPrinting.js [hunk 3]
```diff
@@ -7,11 +8,16 @@ import {
...
 let GENERAL_API_URL = process.env.NODE_ENV === 'production' ?
   config.GENERAL_API_URL_PROD : config.GENERAL_API_URL;
-let LOGGING_API_URL = process.env.NODE_ENV === 'production' ?
-  config.LOGGING_API_URL_PROD : config.LOGGING_API_URL;
+let PERIPHERAL_API_URL = process.env.NODE_ENV === 'production' ?
+  config.PERIPHERAL_API_URL_PROD : config.PERIPHERAL_API_URL;
 let RPC_API_URL = process.env.NODE_ENV === 'production' ?
   config.RPC_API_URL_PROD : config.RPC_API_URL;
 
+  console.log({
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:16]: noooooooooo

File: api/peripheral_api/server.js [hunk 4]
```diff
@@ -9,9 +9,10 @@ function main() {
     __dirname + '/routes/Printer.js'
 
   ];
-  const loggingServer = new SceHttpServer(API_ENDPOINTS, 8081, '/logging_api/');
-  loggingServer.initializeEndpoints().then(() => {
-    loggingServer.openConnection();
+  const peripheralServer = new
+  SceHttpServer(API_ENDPOINTS, 8081, '/peripheral_api/');
```
Comment (evanugarte) [inline api/peripheral_api/server.js:13]: if the prefix u are doing is `peripheral_api` make sure config.example.json matches, i believe is it is `peripheralapi` atm

**Review by evanugarte — CHANGES_REQUESTED** [review]

**Comment by allen-tran** [comment]
grrrrrrrrrr

File: api/peripheral_api/models/PrintLog.js [hunk 5]
```diff
@@ -9,7 +9,6 @@ const PrintLogSchema = new Schema(
     },
     chosenPrinter: {
       type: String,
-      required: true
```
Comment (evanugarte) [inline api/peripheral_api/models/PrintLog.js:12]: removing this for now because core-v4 has no way of knowing which printer was chosen atm

**Review by evanugarte — APPROVED** [review]

**Pull request merged**